test/interpreter/function-vars.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VAR.S', () => {
+  it('should take at least two arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.S()'],
+      ['=VAR.S(1)']
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('A2'))).toEqualError(detailedError(ErrorType.DIV_BY_ZERO))
+  })
+
+  it('should calculate variance (sample)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.S(2, 3)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.5)
+  })
+
+
+  it('should coerce explicit argument to numbers', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.S(2, 3, 4, TRUE(), FALSE(), "1",)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(2.28571428571429, 6) //inconsistency with product #1
+  })
+
+  it('should ignore non-numeric values in ranges', () => {
```

Comment:
Same situtation with VARA and VARS as I've described for VARPS and VARPA.

```suggestion
  it('should ignore non-numeric values in ranges, including ignoring logical values and text representation of numbers', () => {
```